feature/home/presentation/home-ui/src/main/java/com/chocolatecake/ui/tv_details/TvDetailsFragment.kt

```diff
@@ -122,8 +122,11 @@ class TvDetailsFragment :
         }
     }
     //region rating bottom sheet
-    private fun showRateBottomSheet() {
+    private fun showRateBottomSheet(voteAverage: Float) {
         rateBottomSheet = RateBottomSheet()
+        rateBottomSheet.arguments = Bundle().apply {
+            putFloat("voteAverage", voteAverage)
+        }
```

Comment:
try to access the voteAverage from the bottom sheet using the viewModel instead of passing it throught bundle